Directory.Packages.props

```diff
@@ -117,6 +119,7 @@
     <PackageVersion Include="Microsoft.VisualStudio.Web.CodeGeneration.Design" Version="6.0.14" />
     <PackageVersion Include="Moq" Version="4.18.4" />
     <PackageVersion Include="Newtonsoft.Json" Version="13.0.3" />
+    <PackageVersion Include="NRedisStack" Version="0.12.0" />
```

Comment:
do we need this package? I don't see it included in any cs file

Reply:
It was needed before, but I think I added the `StackExchange.Redis` one afterwards that includes it. Thanks for pointing it out, will verify

Reply:
This package is built on top of a redis client package `StackExchange.Redis`. We import the client one when we need to setup the connection, but when we'll be interacting with the cache we'll need to import this package